configure.ac

```diff
@@ -9,6 +9,14 @@ AC_TYPE_SSIZE_T
 AC_FUNC_MALLOC
 AC_FUNC_REALLOC
 AC_CHECK_FUNCS([memmove setlocale strdup])
+AC_SEARCH_LIBS([strlcpy], [bsd], [], [
+  AC_MSG_ERROR([unable to find strlcpy() function])
+])
```

Comment:
If neither `strlcpy` nor libbsd is present on the system, will this error out? #10 bundles implementations of `strlcpy` and `strlcat` which are used if `AC_CHECK_FUNCS` doesn't find the functions. Would that do as a replacement for `AC_SEARCH_LIBS`?